<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorGame</class>
 <widget class="QMainWindow" name="MotorGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NeuroGame - Habilidad motora</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(254, 245, 240);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>57</y>
      <width>581</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 57, 132);</string>
    </property>
    <property name="text">
     <string>Â¡Encuentra el animal!</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>184</y>
      <width>1140</width>
      <height>487</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="button_1">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>33</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_4">
     <property name="geometry">
      <rect>
       <x>274</x>
       <y>33</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_7">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>33</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_13">
     <property name="geometry">
      <rect>
       <x>982</x>
       <y>33</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_11">
     <property name="geometry">
      <rect>
       <x>746</x>
       <y>33</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_8">
     <property name="geometry">
      <rect>
       <x>746</x>
       <y>200</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_9">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>200</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_5">
     <property name="geometry">
      <rect>
       <x>274</x>
       <y>200</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_2">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>200</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_12">
     <property name="geometry">
      <rect>
       <x>746</x>
       <y>367</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_6">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>367</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_3">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>367</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_14">
     <property name="geometry">
      <rect>
       <x>982</x>
       <y>367</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_10">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>367</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_15">
     <property name="geometry">
      <rect>
       <x>982</x>
       <y>200</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="music">
    <property name="geometry">
     <rect>
      <x>1166</x>
      <y>51</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/music.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>33</y>
      <width>67</width>
      <height>66</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/star.png);
color: rgb(221, 122, 179);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timer_text">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 57, 132);</string>
    </property>
    <property name="text">
     <string>timer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sound">
    <property name="geometry">
     <rect>
      <x>1217</x>
      <y>51</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/volume_down.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="random_animal">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>33</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/assets/images/ellipse .png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="question">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>36</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(69, 69, 229);</string>
    </property>
    <property name="text">
     <string>Â¿           ?</string>
    </property>
   </widget>
   <widget class="QLabel" name="number_of_plays">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(221, 122, 179);</string>
    </property>
    <property name="text">
     <string>0 / 10</string>
    </property>
   </widget>
   <zorder>question</zorder>
   <zorder>label</zorder>
   <zorder>frame</zorder>
   <zorder>music</zorder>
   <zorder>score</zorder>
   <zorder>timer_text</zorder>
   <zorder>sound</zorder>
   <zorder>random_animal</zorder>
   <zorder>number_of_plays</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
